Assets/Vimeo/Scripts/Config/VimeoVideo.cs

```diff
@@ -1,6 +1,10 @@
 using System; 
 using System.Text.RegularExpressions;
 using SimpleJSON;
+using System.Collections;
+using System.Collections.Generic;
+using Vimeo.Player;
```

Comment:
We don't need to require `Vimeo.Player` if we are working in the `Vimeo` namespace, right?